apps/condo/domains/acquiring/schema/AcquiringIntegration.js

```diff
@@ -73,6 +73,7 @@ const AcquiringIntegration = new GQLListSchema('AcquiringIntegration', {
             isRequired: true,
         },
 
+        // TODO(pahaz): we should remove this field and use `billingAcquiringGroup` filed!
```

Comment:
`billingAcquiringGroup` is not obvious naming....And it's also good to provide jira ticket here

Reply:
It's not the final version. We need to record some architectural decisions to not forgot it